experiments/where-do-users-find-our-sites.md

```diff
@@ -0,0 +1,103 @@
+# Where do users find out about Alaveteli sites?
+
+Where is our existing traffic being referred from? Where might there be
+potential users that we're not currently reaching?
+
+- [issue#4](https://github.com/mysociety/alaveteli-experiments/issues/4)
+
+## Answer
+
+**TL;DR:** so far, referrer analysis reveals a dominance of social media sites
+for most of our sites with more of a news/campaign-driven audience for the more
+journalistic sites such as Ask The EU. Inbound link analysis inconclusive due to
+the limited features of free reports.
+
+## Analysis
+
+This has turned out to be a more complex question than we first realised so we're
+going to try to update this as we find out more.
+
+We have two types of data available:
+
+### [Referrer data](where-do-users-find-our-sites/inbound-traffic.md)
+
+Data extracted from Google Analytics showing which links the people who are
+visiting our sites have clicked to reach us.
+
+From studying this data we appear to have two types of site - those which get
```

Comment:
I think my site categorisation needs a bit of a shake-up as I suspect part of what we're seeing here is that a lot of early stage traffic for new sites comes from partner orgs before they really take off - for example the "App" traffic looks like test data and is only about 20 hits, all from the same account if I've understood the referrer URL structure correctly.